spark-doris-connector/pom.xml

```diff
@@ -132,6 +134,12 @@
         </profile>
     </profiles>
     <dependencies>
+        <dependency>
```

Comment:
`netty-all` needs to be in front of `arrow-memory-netty`, because `arrow-memory-netty` introduces the `netty-buffer` version, and the wrong version will be loaded during test.

Reply:
spark-core has introduced netty-all dependency, no need to add it here.